images/ipfailover/keepalived/lib/config-generators.sh

```diff
@@ -44,6 +44,8 @@ function generate_global_config() {
 
 #
 #  Generate VRRP checker script configuration section.
+#    When port is 0 always pass
```

Comment:
Actually, upon reflection, I think this is wrong.

If they give us a check script, always use it
If the port is non-zero set up the tcp check
Otherwise, skip the test